src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -71,6 +71,22 @@ message Image {
     string digest = 3;
 }
 
+message NetworkConfiguration {
+    enum NetworkMode {
+        // Provides no network access to a container (Not supported yet)
+        None = 0;
+        // IPv4 only is a sane default for 2001, where task gets a single IPv4 address only
+        ipv4Only = 1;
+        // IPv6 And IPv4 (True Dual Stack)
+        ipv6AndIpv4 = 2;
+        // IPv6 and IPv4 Fallback uses the Titus IPv4 "transition mechanism" to give ipv4
+        // connectivity transparently without providing every container their own IPv4 address
+        ipv6AndIpv4Fallback = 3;
+        // IPv6 Only is for true believers, and won't be a standard networking mode for a while
+        ipv6Only = 4;
+    }
+}
```

Comment:
Do you need to make a NetworkMode field here?

Reply:
A right. Thank you for help here, this is my (second) time ever editing a protobuf